File: 7KYU/Alphabet symmetry/solution.py
def solve(arr):
    alphabet = "abcdefghijklmnopqrstuvwxyz"
    matches = []
    for s in arr:
        count = 0
        for i, e in enumerate(s.lower()): 
            if i == alphabet.find(e):
                count = count + 1
        matches.append(count)
    return matches

File: 7KYU/Alphabet symmetry/solution.clj
(ns kata
  (:require [clojure.string :as str]))

(defn is-alphabetical [idx c]
  (= (nth "abcdefghijklmnopqrstuvwxyz" idx) c))

(defn alphabetical-chars [s]
  (->> s
       str/lower-case
       (map-indexed is-alphabetical)
       (filter true?)
       (count)))

(defn solve [xs]
  (map alphabetical-chars xs))